go.mod

```diff
@@ -2,32 +2,32 @@ module github.com/rancher/steve
 
 go 1.13
 
-replace k8s.io/client-go => k8s.io/client-go v0.17.2
+replace k8s.io/client-go => k8s.io/client-go v0.18.0
 
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/gorilla/context v1.1.1 // indirect
 	github.com/gorilla/mux v1.6.2
 	github.com/gorilla/websocket v1.4.0
 	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/rancher/dynamiclistener v0.2.1-0.20200213165308-111c5b43e932
-	github.com/rancher/wrangler v0.5.3
-	github.com/rancher/wrangler-api v0.5.0
+	github.com/rancher/wrangler v0.5.4-0.20200326191509-4054411d9736
```

Comment:
Need to update this after merging https://github.com/rancher/wrangler/pull/93 with the commit from the release/v2.4 branch